### ISSUE TITLE
Public release?

### ISSUE THREAD

**Issue by drphilmarshall** [yymao/kipac-teabot#19]
Hey @yymao - the KIPAC TeaBot is extremely useful code, and other institutes are showing interest in running it. Can you comment briefly on how hard it would be to prepare it for general release, and how much maintenence you think that would take? Also, you'll need someone at KIPAC to maintain the system after you've gone - could be good to involve that person in the work (documentation, testing etc) leading up to release. One way to absolve yourself of responsibility coudl be to transfer ownership to KIPAC, where the repo would be KIPAC/TeaBot. However, this would diminish your visibility as its creator! I guess we'd just write your name and website all over it :-)

**Comment by yymao** [comment]
Yes, I do plan to make TeaBot publicly available. There are two separate tasks here:

1. Make the feature of personal recommendation publicly available, without the need of the membership of a particular institute. 
2. Make the institute-level features (paper discovery, discussion leaders/topics suggestion, etc) publicly available to other institutes.

For the first task, I need to move this service to a standalone server. For the second one, there are two options to do this:

1. Move the service to a standalone server, and other institutes can subscribe it (like voxCharta). 
2. Revamp this repo to make it easy to install, with details instructions so other institutes can install it on their own machines/servers. (This would involve more work for the other institutes.)

I am still look for an appropriate server to move this project onto. If KIPAC is willing to sponsor me and let this whole service live on KIPAC machines, it would be great. (It's like KIPAC hosting a service for the public + other institutes.) If not, I will then at least move the KIPAC-wide service onto KIPAC machine and keep working the other public-facing one. 

In any case, my current plan is to work on this transition early next year (i.e., 2016 winter). Maybe we (+Tom A, Risa W, Stuart M, and others) should find a time to discussion these options.

**Comment by drphilmarshall** [comment]
Sounds good! Keep us posted :-)
